docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
     links:
```

Comment:
All the docker stuff below could be dropped if we agree that the feature of `markus` to log the statsd calls locally is sufficient. It has a logging backend.

Reply:
Yeah I agree, just using the logging backend is sufficient we don't need to use the echo server 👍